feature/appdetail/src/main/kotlin/com/merxury/blocker/feature/appdetail/AppDetailViewModel.kt

```diff
@@ -366,7 +366,7 @@ class AppDetailViewModel @Inject constructor(
     fun changeSearchMode(isSearchMode: Boolean) {
         Timber.v("Change search mode: $isSearchMode")
         if (!isSearchMode) {
-            loadComponentList()
+            search("")
```

Comment:
What's the difference?

Reply:
loadComponentList() doesn't clear keyword